prow/config/config.go

```diff
@@ -34,6 +34,13 @@ type Config struct {
 
 	// Periodics are not associated with any repo.
 	Periodics []Periodic `json:"periodics,omitempty"`
+
+	Triggers []Trigger `json:"trigger,omitempty"`
+}
+
+type Trigger struct {
+	Repos      []string `json:"repos,omitempty"`
+	TrustedOrg string   `json:"trusted_org,omitempty"`
```

Comment:
Can you add some comments clarifying the relationship between a TrustedOrg and the Repos? Can we have more than one TrustedOrg? Can we have Repos outside the TrustedOrg?